changes/1260.breaking.md

```diff
@@ -0,0 +1,7 @@
+Support v10 attachments
+
+This includes breaking changes, features and things to look out for:
+- Modifying attachments in messages that contain embeds with any image attached to them now requires the images of that embed
+  image to be re-passed in the edit or they will be lost.
+- `replace_attachments` has been removed, as it is now the default.
+  - `attachment` and `attachments` now supports `None` to replicate the behaviour of fully removing all attachments.
```

Comment:
```suggestion
  - `attachment` and `attachments` now supports `None` to replicate the behaviour of fully removing all attachments.
- `attachment` and `attachments` in message edits now support passing an `Attachment` object to keep existing attachments.
```

Note to self: Search replace affected message create too. Go ahead and revert those